app/policies/comment_policy.rb

```diff
@@ -0,0 +1,17 @@
+# Pundit class to control authorization for comment actions.
+class CommentPolicy < ApplicationPolicy
+  # only signed up users can comment a book
+  def create?
+    user.present?
+  end
+
+  # only signed up users can update a comment
+  def update?
+    user.present?
```

Comment:
Any user can delete any comment? that doesn't make much sense, only the author of the comment or an admin can update a comment